documentation/Tutorials/WPF/Categories1.md

```diff
@@ -85,7 +85,7 @@ namespace Caliburn.Micro.Tutorial.Wpf.ViewModels
       get => _categoryDescription; set
         {
         _categoryDescription = value;
-        NotifyOfPropertyChange(()=>CategoryDescription);
+        NotifyOfPropertyChange(nameof(CategoryDescription));
```

Comment:
you could just use NotifyOfPropertyChange();

The method will get the caller name

public virtual void NotifyOfPropertyChange([System.Runtime.CompilerServices.CallerMemberName] string propertyName = null)

Reply:
Thank you for this addition. I added it to the text.